data-prepper-api/src/main/java/org/opensearch/dataprepper/model/annotations/DataPrepperPlugin.java

```diff
@@ -46,6 +48,12 @@
      */
     String deprecatedName() default DEFAULT_DEPRECATED_NAME;
 
+    /**
+     *
+     * @return Alternate name of the plugin which should be unique for the type
+     */
+    String alternateName() default DEFAULT_ALTERNATE_NAME;
```

Comment:
Let's make this an array. I could see this growing over time.

```
String[] alternateNames()
```

Reply:
Done